crossdock/server/server.go

```diff
@@ -77,6 +85,9 @@ func (s *Server) Start() error {
 			return s.doJoinTrace(ctx, req.(*tracetest.JoinTraceRequest))
 		})
 	})
+	// TODO test_driver host needs to be initialized before this
+	//mux.HandleFunc("/endtoend", s.eHandler.Trace)
```

Comment:
the http server panics since the test_driver hostname in the config doesn't exist yet